### PR TITLE
Create ttp [Retrieve in Memory Passwords]

### PR DESCRIPTION
**Pull request by abhiabhi2306**
This technique can be used to retrieve passwords from the main memory character device image file (/dev/mem)

### CODE REVIEW

File: discovery/0e016565-f44f-4735-bfec-b714b87ee4b7.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+id: 0e016565-f44f-4735-bfec-b714b87ee4b7
+metadata:
+  version: 1
+  authors: 
+    - Abhishek S (abhiabhi2306)
+  tags: []
+name: Retrieve in Memory Passwords.
```
Comment (khyberspache) [inline discovery/0e016565-f44f-4735-bfec-b714b87ee4b7.yml:7]: remove the period and lowercase `memory passwords`

**Review by khyberspache — CHANGES_REQUESTED** [review]

**Comment by abhiabhi2306** [comment]
Oops, sorry, thank you for bringing that to my attention, rectified!

Kind Regards,
Abhi

File: discovery/0e016565-f44f-4735-bfec-b714b87ee4b7.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+id: 0e016565-f44f-4735-bfec-b714b87ee4b7
+metadata:
+  version: 1
+  authors: 
+    - Abhishek S (abhiabhi2306)
+  tags: []
+name: Retrieve in memory passwords
+description: |
+  This technique can be used to retrieve passwords from the main memory character device image file (/dev/mem)
+technique:
```
Comment (khyberspache) [inline discovery/0e016565-f44f-4735-bfec-b714b87ee4b7.yml:10]: missing tactic

**Review by khyberspache — CHANGES_REQUESTED** [review]

**Comment by abhiabhi2306** [comment]
Fixed, thank you ^^

**Review by khyberspache — APPROVED** [review]

**Pull request merged**